mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -758,6 +758,17 @@ void mav_vehicle::rotate(double angle_deg, bool autocontinue)
     // Update timestamp
```

Comment:
comments are all over the place. I mean merge "Continue to detour or mission waypoint after brake" with "Continue to detour or mission waypoint after rotating"